test/e2e/block_sync_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 the orbs-network-go authors
+// This file is part of the orbs-network-go library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package e2e
+
+import (
+	"github.com/orbs-network/orbs-network-go/test"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestBlockSyncRecover(t *testing.T) {
```

Comment:
there are some hidden assumptions here - that the node we get the metrics from is the node that did not receive his blocks file.. can you please  open an issue to make it clearer or add some comments?